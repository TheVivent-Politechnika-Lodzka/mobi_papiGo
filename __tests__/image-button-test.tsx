import 'react-native';
import React from 'react';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';
import ImageButton from '../src/components/image-button';

const img = require('../assets/logo.png');

it('Button render test', () => {
  const button = renderer
    .create(<ImageButton src={img} onPress={() => {}} />)
    .toJSON();
  expect(button).toMatchSnapshot();
});
