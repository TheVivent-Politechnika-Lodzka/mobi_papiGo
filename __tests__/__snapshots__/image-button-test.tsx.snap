// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button render test 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    accessible={true}
    collapsable={false}
    focusable={true}
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#C8E3D4",
        "borderRadius": 10,
        "elevation": 10,
        "height": "80%",
        "justifyContent": "center",
        "shadowColor": "#000",
        "shadowOffset": Object {
          "height": 4,
          "width": 4,
        },
        "shadowOpacity": 0.5,
        "shadowRadius": 10,
        "width": "80%",
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../assets/logo.png",
        }
      }
      style={
        Object {
          "height": "90%",
          "width": "90%",
        }
      }
    />
  </View>
</View>
`;
